packages/x-charts/src/internals/components/AxisSharedComponents.tsx

```diff
@@ -63,6 +64,6 @@ export const ChartsLabel = styled('text', {
   slot: 'Label',
   overridesResolver: (props, styles) => styles.label,
 })(({ theme }) => ({
-  ...theme.typography.body1,
+  ...(theme.typography as Record<string, CSSObject>).body1,
```

Comment:
Maybe also add the comment here? 🤔 
What do you think about also adding a `TODO` of some sort in order to not forget about it eventually? 🤔

Reply:
That's an error, this modification should not be here

Reply:
My apologies, I did not pushed the last commit which revert all those change due to the TS error I'm talking about in the PR description